electron_app/src/electron-main.js

```diff
@@ -321,15 +322,15 @@ app.on('ready', () => {
         height: mainWindowState.height,
         webPreferences: {
             preload: preloadScript,
-            nodeIntegration: false,
             sandbox: true,
-            enableRemoteModule: false,
+            nodeIntegration: false,
```

Comment:
This one too? :)

Reply:
enableRemoteModule: false leads to errors in my case. Can I leave it?

Reply:
![image](https://user-images.githubusercontent.com/6655915/60774503-fac2d080-a114-11e9-8bf6-070ab9eb010b.png)

Reply:
@anoadragon453 ?

Reply:
Sorry, I meant just moving it back to where it was originally in the list, not changing the value.

Reply:
fixed